src/plugins/intel_cpu/tests/functional/single_layer_tests/augru_sequence.cpp

```diff
@@ -0,0 +1,391 @@
+      { {2, 1, 1},                                  // Dynamic shape 1
+        { {2, 1, 1}, {2, 1, 1}, {2, 1, 1} } },      // Target shapes
+      { {2, {1, 5}, 1},                             // Dynamic shape 2
+        { {2, 2, 1}, {2, 3, 1}, {2, 4, 1} } } },    // Target shapes
+    { { {5, -1, 10},                                // #7. Dynamic shape 0
+        { {5, 2, 10}, {5, 4, 10}, {5, 5, 10} } },   // Target shapes
+      { {5, 1, 10},                                 // Dynamic shape 1
+        { {5, 1, 10}, {5, 1, 10}, {5, 1, 10} } },   // Target shapes
+      { {5, -1, 1},                                 // Dynamic shape 0
+        { {5, 2, 1}, {5, 4, 1}, {5, 5, 1} } } },    // Target shapes
+    { { {{0, 11}, -1, {7, 11}},                     // #8. Dynamic shape 0
+        { {10, 2, 10}, {3, 4, 10}, {5, 5, 10}, {10, 2, 10}, {5, 5, 10} } },  // Target shapes
+      { {-1, 1, {8, 12}},                           // Dynamic shape 1
+        { {10, 1, 10}, {3, 1, 10}, {5, 1, 10}, {10, 1, 10}, {5, 1, 10} } },  // Target shapes
+      { {{0, 11}, -1, 1},                           // Dynamic shape 3
+        { {10, 2, 1}, {3, 4, 1}, {5, 5, 1}, {10, 2, 1}, {5, 5, 1} } },     // Target shapes
+      { {-1},                                       // Dynamic shape 2
+        { {10}, {3}, {5}, {10}, {5} } } }            // Target shapes
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_dynamic, AUGRUSequenceCPUTest,
+            ::testing::Combine(::testing::ValuesIn({dynamicShapes[0], dynamicShapes[1], dynamicShapes[2]}),
+                               ::testing::ValuesIn(mode),
+                               ::testing::ValuesIn(activations),
+                               ::testing::ValuesIn(clip),
+                               ::testing::ValuesIn(linearBeforeReset),
+                               ::testing::ValuesIn(direction),
+                               ::testing::ValuesIn(netPrecisions),
+                               ::testing::Values(cpuParams),
+                               ::testing::Values(std::map<std::string, std::string>{})),
+            AUGRUSequenceCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_dynamic_BatchSizeOne, AUGRUSequenceCPUTest,
+            ::testing::Combine(::testing::ValuesIn({dynamicShapes[4]}),
+                               ::testing::ValuesIn(mode),
+                               ::testing::ValuesIn(activations),
+                               ::testing::ValuesIn(clip),
+                               ::testing::ValuesIn(linearBeforeReset),
+                               ::testing::ValuesIn(direction),
+                               ::testing::ValuesIn(netPrecisions),
+                               ::testing::Values(cpuParamsBatchSizeOne),
+                               ::testing::Values(std::map<std::string, std::string>{})),
+            AUGRUSequenceCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(nightly_dynamic, AUGRUSequenceCPUTest,
```

Comment:
@nshchego What‘s the "nightly_dynamic" test suite trying to cover? It looks quite similar to "smoke_dynamic" to me. It's copied from gru_sequence unit test.

Reply:
These tests were divided to reduce precommit scope.

Reply:
I see. Thanks.